spec/model/food_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
```

Comment:
- [ ] When I try to run your tests, I get an error `Failure/Error: require 'rails_helper'`. This happens because you didn't install `RSpec`. Kindly install `RSpec` by running this code in your terminal `rails generate rspec:install` so that it can generate `rails_helper.rb` and `spec_helper.rb` files. 👍

Reply:
![Screenshot from 2022-05-20 00-56-28](https://user-images.githubusercontent.com/52419124/169422719-d66bba2c-27fc-4b14-b519-e53e7c4d7123.png)


I did and all my test passed